server/routers/outfit.py

```
@@ -207,16 +211,32 @@ def _score_and_parse_outfits(
             if result.overall < 0.4:
                 continue
+            event_payload = outfit.get("event")
+            event_ref: OutfitEventRef | None = None
+            if isinstance(event_payload, dict) and event_payload.get("title"):
+                try:
+                    event_ref = OutfitEventRef(
+                        title=event_payload.get("title", ""),
+                        start_time=event_payload.get("start_time", ""),
+                        classified_type=event_payload.get("classified_type", ""),
+                    )
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)